src/graphql/mirror.js

```diff
@@ -728,6 +740,68 @@ export class Mirror {
     }
   }
 
+  /**
+   * Save the request query and query parameters to the database.
+   */
+
+  async _log_request(
```

Comment:
nit: No blank line between doc comment and function declaration.

Reply:
updated